### ISSUE TITLE
Access to new game button is too difficult

### ISSUE THREAD

**Issue by Tomin1** [Tomin1/patience-deck#10]
Currently starting a new game is only possible from Options & Rules page but that requires using pulley menu to first open that page. In contrast Restart button requires only a press of a button on toolbar. I did not want to add new game button to toolbar since it is destructive, i.e. player cannot get back to the previous game if they press new game accidentally. This needs some thought to not break this rule while allowing easier access to new game action.

**Comment by llewelld** [comment]
For me, the location of the "New game" button feels unintuitive, and it's also the more useful of the two buttons for me. One suggestion for working around the destruction issue might be to add a remorse timer.

**Comment by Tomin1** [comment]
This came up when I discussed the game with another friend of mine. It's probably a good idea for me to improve this next.

> One suggestion for working around the destruction issue might be to add a remorse timer.

I think this is a good idea. Additionally I could probably keep the state of the previous game in Recorder a bit longer so that new game can be started immediately but the previous game can be still restored if needed. This improvement could also apply to "Restart" button which is also destructive even if it leaves game seed untouched.

**Comment by Tomin1** [comment]
> Additionally I could probably keep the state of the previous game in Recorder a bit longer so that new game can be started immediately but the previous game can be still restored if needed.

One more thing to consider: Recorder isn't the one that stores game options, so keeping multiple Recorder states is not enough if options can change which also starts a new game. Alternatively, I could also just limit this to restart and new game buttons on toolbar, but I suppose having the same kind of remorse timer when selecting another game or changing options would be nice.

Now that I actually read remorse timer documentation it seems that the expected behaviour is to delay the action. Perhaps that is what I should do. That ought to be quite simple to do.

There is however an alternative idea that I've also thought about. With restart and new game buttons I could keep the previous state and start the new game immediately, but keep the previous state behind undo button. Pressing that before making any moves would restore the previous state. (And leave changing options and selecting another game behind remorse timer for example.) I think I just have to implement both and see what I like most.

**Comment by Tomin1** [comment]
I didn't like remorse timers in connection with toolbar very much. I think the alternative approach works much better. Adding a remorse timer takes that immediacy of toolbar buttons away and that feels wrong. To make it clear that undo can be also used for getting back to the previous game, I [created some new icons](https://github.com/Tomin1/patience-deck/commit/d4494c80494d22e05defbb1111365d06a787f47e) and undo button will change its looks when it functions like that.

This has been now implemented in master branch with commit 7bf5c056c8f7f5037a6203005d52b3b0d401e3cc (and the commits before it). I wrote most of my reasonings for different ideas into the commit messages. I may still want to adjust the icons slighty, but that will be in another commit.
